nautobot/extras/tests/test_dynamicgroups.py

```diff
@@ -327,6 +327,25 @@ def test_map_filter_fields(self):
         self.assertNotIn("q", fields)
         self.assertIn("name", fields)
 
+    def test_map_filter_fields_skip_missing(self):
+        """
+        Test that missing fields are skipped n `DynamicGroup._map_filter_fields`
+        when `Model.dynamic_group_skip_missing_fields` is `True`.
+        """
+        group = self.groups[0]
+
+        try:
+            group.model.dynamic_group_skip_missing_fields = True
+            fields = group._map_filter_fields
```

Comment:
It's a bit weird that this is a property, given its name I would have expected it to be a method. But I can certainly see the benefit of allowing it to be cached.

Reply:
Yeah, it's so that it can be cached as a `@cached_property`. The method `get_filter_fields()` calls this.